src/devices/Ssd13xx/Commands/ISharedCommand.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Ssd13xx.Commands
+{
+    public interface ISharedCommand : ICommand
```

Comment:
consider adding a bool flag to the ICommand (i.e. IsShared) instead

Reply:
The purpose of using ISharedCommand is to make type safe commands executions. Some commands may be executed by both devices bindings. However some commands maybe executed only by specific bindings. Using typed commands allows us to evaluate commands compatibility on compile time.